<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="welcomePage">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>0</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="searchLayout">
       <item>
        <widget class="QLineEdit" name="searchLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>247</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="usernameLabel">
           <property name="text">
            <string>Username :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="usernameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="passwordLabel">
           <property name="text">
            <string>Password : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLineEdit">
           <property name="inputMethodHints">
            <set>Qt::ImhHiddenText</set>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="submitButton">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="registerPushButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>New Student Register</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="secyMainPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Department Secretary's Page</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>296</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="secyToolsLayout">
       <item>
        <widget class="QPushButton" name="enterRegistrationDetailsButton">
         <property name="text">
          <string>Key in Registration Details</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="manageProjectsButton">
         <property name="text">
          <string>Manage Research Projects</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="managePublicationsButton">
         <property name="text">
          <string>Manage Publications/Research Papers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="manageInventoryButton">
         <property name="text">
          <string>Manage Inventory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="manageAccountsButton">
         <property name="text">
          <string>Manage Accounts</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="helloLabel">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>24</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hello, Secretary</string>
      </property>
     </widget>
     <widget class="QPushButton" name="secretaryLogoutPushButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ResearchProjectsPage">
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>502</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="ReseachPaperToolsLayout">
       <item>
        <widget class="QPushButton" name="AddResearchPaperPushButton">
         <property name="text">
          <string>Add Research Paper</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ViewResearchPaperPushButton">
         <property name="text">
          <string>View Research Papers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteResearchPaperPushButton">
         <property name="text">
          <string>Delete Research Paper</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="ResearchPaperStackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>601</width>
        <height>341</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="ResearchProjectWelcome">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>491</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Add/View/Remove Research Papers by clicking on the options above.</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ResearchProjectAdd">
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>441</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="PaperNameLineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="JournalLabel">
           <property name="text">
            <string>Journal </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="AuthorLabel">
           <property name="text">
            <string>Author </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="AbstractTextEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="AuthorLineEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="PaperNameLabel">
           <property name="text">
            <string>Paper Name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="JournalLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="AbstractLabel">
           <property name="text">
            <string>Abstract </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QTableView" name="papersListTableView">
        <property name="geometry">
         <rect>
          <x>75</x>
          <y>11</y>
          <width>491</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="MorePapersPushButton">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>290</y>
          <width>98</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>More Papers</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6"/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
